packages/turbo-workspaces/src/managers/npm.ts

```diff
@@ -57,9 +57,9 @@ async function read(args: ReadArgs): Promise<Project> {
       lockFile: "package-lock.json",
     }),
     workspaceData: {
-      globs: packageJson.workspaces || [],
+      globs: (packageJson.workspaces as Array<string> | undefined) || [],
       workspaces: expandWorkspaces({
-        workspaceGlobs: packageJson.workspaces,
+        workspaceGlobs: packageJson.workspaces as Array<string> | undefined,
```

Comment:
`npm` also supports declaring workspaces via `"workspaces": { "packages": ["package/a"] }` so we should use the `parseWorkspacePackages` method you added. [source](https://github.com/npm/map-workspaces/blob/main/lib/index.js#L29-L40)